iterreg/sparse/solvers.py

```diff
@@ -5,7 +5,8 @@
 from scipy import sparse
 from scipy.sparse.linalg import svds
 
-from iterreg.utils import shrink, ell1
+from iterreg.utils import shrink, power_method
+from nonconvex.prox_functions import select_penalty, select_prox, prox_MCP, prox_MCP_scalar, MCP, select_prox_sc
```

Comment:
you import these from another package (ènonconvex`) which is not a dependency : it will fail 

you need to be more flexible and pass `prox` and `penalty` to the solvers (but I have done it in master already): 

https://github.com/LCSL/iterreg/commit/5328b5465b997265146e1d07eba23d57722d8dc1